### ISSUE TITLE
Unable to add version 5.2.8 for angular-dialog-service

### ISSUE THREAD

**Issue by vgekhman** [webjars/webjars#1168]
Failed! Server returned HTTP response code: 503 for URL: https://bower-as-a-service.herokuapp.com/download/angular-dialog-service/5.2.8 If you feel you have reached this failure in error, please file an issue: https://github.com/webjars/webjars/issues  

Generated POM 
Converted dependencies to Maven 
Got Bower info 
Determined Artifact Name: angular-dialog-service 
Starting Deploy

**Comment by jamesward** [comment]
When I run `bower install angular-dialog-service#5.2.8` locally I get:
```
Unable to find a suitable version for angular, please choose one:
    1) angular#>=1 <1.3.0 which resolved to 1.2.28 and is required by angular-bootstrap#0.12.1
    2) angular#>=1.2.26 <=1.5 which resolved to 1.2.28 and is required by angular-translate#2.7.2
    3) angular#^1.2 which resolved to 1.4.4 and is required by angular-dialog-service#5.2.8
    4) angular#1.4.4 which resolved to 1.4.4 and is required by angular-sanitize#1.4.4
```

I'm not sure how we can deal with the conflicting transitive deps in an automated way.  Any ideas?

**Comment by jamesward** [comment]
This has been fixed: https://github.com/jamesward/bower-as-a-service/commit/caf24d097fdd586a1003195609d6197b80c43419#diff-0364f57fbff2fabbe941ed20c328ef1aR55
